client/profile-wizard/steps/product-types.js

```diff
@@ -13,13 +13,35 @@ import { withDispatch, withSelect } from '@wordpress/data';
  * WooCommerce dependencies
  */
 import { getSetting } from '@woocommerce/wc-admin-settings';
-import { H, Card, Link } from '@woocommerce/components';
+import { H, Card, Link, Pill } from '@woocommerce/components';
 import { ONBOARDING_STORE_NAME } from '@woocommerce/data';
 
 /**
  * Internal dependencies
  */
 import { recordEvent } from 'lib/tracks';
+import './product-types.scss';
+
+function getLabel( description, yearlyPrice ) {
+	if ( ! yearlyPrice ) {
+		return description;
+	}
+
+	const monthlyPrice = ( yearlyPrice / 12.0 ).toFixed( 2 );
+	const priceDescription = sprintf(
+		'$%f per month, billed annually',
```

Comment:
Looks like we missed a translation wrapper on this text.

Reply:
Whoops, I thought that since `sprintf` came from the i18n library it included translation. Fixed in 6156c3fd24f729209be9f726284ffc98be5ffadf.